src/configure.js

```diff
@@ -70,7 +70,7 @@ function configure ({
       })
 
       try {
-        forwardRequestToNodeServer({
+        await forwardRequestToNodeServer({
```

Comment:
Does this need to be async/await?

Reply:
When I integate with Nestjs, the async/await help me handle exception from Nestjs code.

Reply:
I reverted